### PR TITLE
OCPBUGS-35498 adding clarification from SME

### PR DESCRIPTION
**Pull request by kquinn1204**
[OCPBUGS-35498]: Adding clarification associated with multi-network policies

Version(s): 4.15, 4.16 and main

Issue:https://issues.redhat.com/browse/OCPBUGS-35498

Link to docs preview:https://77710--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-multi-network-policy.html

QE review:
- [x] QE has approved this change.

Additional information: Follow up to https://github.com/openshift/openshift-docs/pull/77630

### CODE REVIEW

**Comment by kquinn1204** [comment]
@zeeke Can you add a LGTM if happy with the change

**Comment by zeeke** [comment]
/lgtm

thanks @kquinn1204

File: networking/multiple_networks/configuring-multi-network-policy.adoc [hunk 1]
```diff
@@ -6,7 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you can configure multi-network for additional networks. You can specify multi-network policy for SR-IOV, macvlan, and OVN-Kubernetes additional networks. Macvlan additional networks are fully supported. Other types of additional networks, such as ipvlan, are not supported.
+As a cluster administrator, you can configure multi-network for additional networks. You can specify multi-network policy for SR-IOV, macvlan, and OVN-Kubernetes additional networks. Macvlan additional networks are fully supported. Other types of additional networks, such as ipvlan, are not supported.
```
Comment (dfitzmau) [inline networking/multiple_networks/configuring-multi-network-policy.adoc:9]:
```suggestion
As a cluster administrator, you can configure a multi-network policy for a Single-Root I/O Virtualization (SR-IOV), MAC Virtual Local Area Network (MacVLAN), or OVN-Kubernetes additional networks. MacVLAN additional networks are fully supported. Other types of additional networks, such as IP Virtual Local Area Network (IPVLAN), are not supported. 
```

File: networking/multiple_networks/configuring-multi-network-policy.adoc [hunk 2]
```diff
@@ -6,7 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you can configure multi-network for additional networks. You can specify multi-network policy for SR-IOV, macvlan, and OVN-Kubernetes additional networks. Macvlan additional networks are fully supported. Other types of additional networks, such as ipvlan, are not supported.
+As a cluster administrator, you can configure multi-network for additional networks. You can specify multi-network policy for SR-IOV, macvlan, and OVN-Kubernetes additional networks. Macvlan additional networks are fully supported. Other types of additional networks, such as ipvlan, are not supported. 
+
+[NOTE]
+====
+Support for configuring multi-network policies for SR-IOV additional networks is only supported with kernel network interface cards (NICs). SR-IOV is not supported for Data Plane Development Kit (DPDK) applications.
```
Comment (dfitzmau) [inline networking/multiple_networks/configuring-multi-network-policy.adoc:13]: NIC = network interface controller?

Screenshot from 2024-06-21 12-04-46

**Comment by dfitzmau** [comment]
Added comments inline. Please correct before adding a merge review label.

/remove-label peer-review-in-progress

/label peer-review-done

**Pull request merged**